lib/hacluster.pm

```diff
@@ -974,49 +976,55 @@ Calculates start time delay after node is fenced.
 Prevents cluster failure if fenced node restarts too quickly.
 Delay time is used either if specified in sbd config variable "SBD_DELAY_START"
 or calculated:
-"corosync_token + corosync_consensus + SBD_WATCHDOG_TIMEOUT * 2"
+"corosync token timeout + consensus timeout + pcmk_delay_max + msgwait"
 Variables 'corosync_token' and 'corosync_consensus' are converted to seconds.
+For diskless SBD pcmk_delay_max is set to static 30s.
```

Comment:
Does it makes sense to describe the expected `sbd_parameters` structure here?

Reply:
Definitely. I just updated the code.